src/main/java/training/commands/ArmCommands/elbowCommands/ElbowDefaultCommand.java

```diff
@@ -0,0 +1,21 @@
+package training.commands.ArmCommands.elbowCommands;
+
+import edu.wpi.first.wpilibj2.command.Command;
+import training.subsystems.ArmSubsystems.elbowSubsystem.Elbow;
+
+public class ElbowDefaultCommand extends Command {
```

Comment:
```suggestion
public class ElbowStayInPlace extends Command {
```
dont call ElbowDefaultCommand, names should say purpose  and not usage